### PR TITLE
Reverse mode for throttle track

### LINKED ISSUE #1159: Add a reverse option to the throttle track.
Selfy explanatory.

This one's on me.

### PR DESCRIPTION
**Pull request by Cream-Tea**
I have added a reverse mode functionality to the throttle track, since it doesn't seem to be a feature at the moment, yet the old limiter track used to have a reverse mode.

Closes #1159

### CODE REVIEW

File: src/main/java/mods/railcraft/common/blocks/tracks/outfitted/kits/TrackKitThrottle.java [hunk 1]
```diff
@@ -40,14 +41,24 @@ public TrackKits getTrackKitContainer() {
...
-        if (isPowered())
-            state += LocoSpeed.VALUES.length;
+        if (getReverse())
+        	state = 14;
+        if (isPowered()) {
+            if (getReverse())
+            	state += 1;
+            else
+            	state += 7;
+    	}
```
Comment (CovertJaguar) [inline src/main/java/mods/railcraft/common/blocks/tracks/outfitted/kits/TrackKitThrottle.java]: You can simplify this by reordering the states.

File: src/main/resources/assets/railcraft/blockstates/tracks/outfitted/kit/throttle.json [hunk 2]
```diff
@@ -86,6 +86,17 @@
...
+        "textures": {
+          "layer0": "railcraft:blocks/tracks/outfitted/kit/limiter_r",
+        }
+      }
+      "15": {
+        "textures": {
+          "layer0": "railcraft:blocks/tracks/outfitted/kit/limiter_r",
+          "layer1": "railcraft:blocks/tracks/outfitted/kit/throttle_overlay_active"
+        }
+      }
```
Comment (CovertJaguar) [inline src/main/resources/assets/railcraft/blockstates/tracks/outfitted/kit/throttle.json]: The six "limiter" texture that never got renamed are intended for reverse mode. If you wanted to add more reverse speeds, you could do it by reintegrating these textures.

**Review by CovertJaguar — COMMENTED** [review]
Just so everyone is on the same page, perhaps some history in order. The track was intended originally to have seven speed modes, three of which were to be reverse. And this was true, until I changed locomotives to handle speed and direction separately. When I did this, the Throttle track lost its reverse capability because the LocoSpeed enum was now only four elements long instead of seven/eight. So if we were to do a proper pass at this, we should clean up those unused reverse "limiter" textures, either rename them to bring them in line with the existing textures and use them, or just delete them entirely and remove the entries from the blockstate JSON.

File: version/gradle.properties [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 version_forestry=[5.1.15.295,)
-version_industrialcraft=[2.6.11-ex110,2.7.0-ex111)
+version_industrialcraft=2.6.188-ex110
 version_project_major=10
 version_project_series=1
 version_project_revision=0
-version_project_build=0
-version_project_qualifier=none
+version_project_build=3.1
+version_project_qualifier=beta
```
Comment (CovertJaguar) [inline version/gradle.properties]: You'll need to revert these two lines.

File: version/gradle.properties [hunk 4]
```diff
@@ -1,11 +1,11 @@
-#Thu, 20 Apr 2017 11:30:12 -0700
+#13:17 21/04/2017
 version_minecraft=1.10.2
 version_forge=12.18.3.2239
 version_mcp=snapshot_20160826
 version_forestry=[5.1.15.295,)
-version_industrialcraft=[2.6.11-ex110,2.7.0-ex111)
+version_industrialcraft=2.6.188-ex110
```
Comment (CovertJaguar) [inline version/gradle.properties]: I could have sworn the existing bounds should have resolved this issue.

**Review by CovertJaguar — CHANGES_REQUESTED** [review]

File: version/gradle.properties (hunk 3 above)
Comment (Cream-Tea) [reply version/gradle.properties]: Alright I'll revert them, I wasn't really too sure about what they were supposed to be.

**Review by CovertJaguar — APPROVED** [review]

**Pull request merged**